Vector2D.java

```diff
@@ -0,0 +1,66 @@
+package org.firstinspires.ftc.teamcode;
+
+/**
+ * Created by NoahR on 9/16/17.
+ */
+
+public class Vector2D {
+    private double m_nXComponent;
+    private double m_nYComponent;
+
+
+    public Vector2D( double iXComponent, double iYComponent ) {}
```

Comment:
Maybe set fields in this constructor, so the vector is initialized.

Reply:
Lol whoops.